group_project/group_project.py

```diff
@@ -244,6 +246,10 @@ def student_view(self, context):
             "ta_graded": (self.group_reviews_required_count < 1),
         }
 
+        user_prefs = self.project_api.get_user_preferences(self.user_id)
+        for activity_component in group_activity.activity_components:
+            activity_component.grading_override = True if "TA_REVIEW_WORKGROUP" in user_prefs else False
```

Comment:
@marjev is these three lines the only actual change here? Do I understand right that if TA is a group member it can access both normal GP view and TA grading view?